packages/core/__tests__/diff/directive.test.ts

```diff
@@ -173,13 +173,13 @@ describe('directive', () => {
       };
 
       // Nullable
-      expect(change.a.criticality.level).toEqual(CriticalityLevel.NonBreaking);
-      expect(change.a.type).toEqual('DIRECTIVE_ARGUMENT_ADDED');
-      expect(change.a.message).toEqual(`Argument 'name' was added to directive 'a'`);
+      expect(change.a?.type).toEqual('DIRECTIVE_ARGUMENT_ADDED');
+      expect(change.a?.criticality.level).toEqual(CriticalityLevel.NonBreaking);
```

Comment:
In several places I reordered things because when tweaking behavior and testing, t was not providing me enough insights to understand what was wrong. Placing the "type" first let me know which even type was being returned.